samples/Http2SampleApp/Program.cs

```diff
@@ -31,7 +29,8 @@ public static void Main(string[] args)
 
                     options.Listen(IPAddress.Any, basePort, listenOptions =>
                     {
-                        listenOptions.Protocols = HttpProtocols.Http1AndHttp2;
+                        // listenOptions.Protocols must be made public to run this sample.
```

Comment:
Should we make InternalsVisibleTo to this sample like we do for our test projects?

Reply:
I'd rather not. In theory this is a temporary change until we start 2.2.

Reply:
I'd rather just leave a comment saying this only works because of InternalsVisableTo. I don't like having to modify the sample just to try out an HTTP/2 endpoint.